outlines/packages.md

```diff
@@ -0,0 +1,32 @@
+# Building a package
+
+1. Why you might want to build a package
+    1. You have a package-based app
+    2. You want to publish some code for people to use in the community
+    3. You want to share code within your organization between different apps
+2. Creating a package using the command line tool
+3. Parts of a Meteor package
+    1. What different parts of package.js mean, point to the docs
+    2. Depending on other packages and build plugins
+         1. You can't depend on a specific version of Meteor; that's not what versionsFrom does
+    3. Adding files and assets
+    4. About architectures
+    5. Semver and the constraint solver
+    6. Cordova plugins
+    7. NPM packages
```

Comment:
> Hmm, is that package-specific? I guess it's just related to using NPM stuff, right?

I just now that @avital told me when packaging came out that this is the main point of why making a Meteor package differs from just using `meteorhacks:npm`. That you should create Fiber-enabled versions. I think this is not enough promoted and then people make automatic publishing of packages and do not make any effort to make Fiber-enabled versions. As a consequence people then get a big mess of Fiber-code, callbacks, and `bindEnvironment`. When all that could be taken care of from package maintainer. So putting this into the official guide would make some pressure on package maintainers to start doing that. People would be able to demand that from them.